Backend/internal/auth/auth_DB.go

```diff
@@ -0,0 +1,53 @@
+// TITLE: Login functions
+// Created by (Jacky: FafnirZ) (09/21)
+// Last modified by (Jacky: FafnirZ)(04/09/21)
+// # # #
+/*
+This module handles the database query and parsing of data for the login functions
+as well as some error handling
+**/
+package auth
+
+import (
+	"DiffSync/database"
+	"context"
+	"log"
+
+	_ "github.com/lib/pq"
+)
+
+var httpDbPool database.Pool
+
+func init() {
+	var err error
+	httpDbPool, err = database.NewPool(database.Config{
+		HostAndPort: "db:5432",
+		User:        "postgres",
+		Password:    "postgres",
+		Database:    "test_db",
+	})
+
+	if err != nil {
```

Comment:
Where do these print statements end up? Should we maybe be implementing exceptions instead?

Reply:
the print statements end up in the console, but ye I could look into that

Reply:
it might be beneficial to have a config validator that runs prior to the backend accepting any requests, the validator can just checks that the provided DB config details actually results in a working connection